cves/CVE-2017-7675.yml

```diff
@@ -4,33 +4,33 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Tomcat URL Traverrsal
```

Comment:
Typo in `Traverrsal`, should be `Traversal`

Reply:
Good catch! :+1: